SDL/Triplet.h

```diff
@@ -349,7 +349,7 @@ namespace SDL {
     float drt_InSeg = rtMid - rtIn;
     float dz_InSeg = zMid - zIn;
     float dr3_InSeg =
-        alpaka::math::sqrt(acc, rtMid * rtMid + zMid * zMid) - alpaka::math::sqrt(acc, rtIn * rtIn + zIn + zIn);
+        alpaka::math::sqrt(acc, rtMid * rtMid + zMid * zMid) - alpaka::math::sqrt(acc, rtIn * rtIn + zIn * zIn);
```

Comment:
this would can potentially affect BBB T3s: the value of `dr3_InSeg` was slightly larger with the bug, which meant a slightly larger z selection window based on the track direction due to a larger component accounting for the multiple scattering. Realistically only T3s starting in barrel layer 1 may show something, but it looks like it was not visible even there.